lib/storage/file.js

```diff
@@ -387,17 +387,18 @@ File.prototype.move = function(destination, callback) {
  * //-
  * var fs = require('fs');
  * var myBucket = gcs.bucket('my-bucket');
- * var image = myBucket.file('image.png');
+ * var remoteFile = myBucket.file('image.png');
+ * var localFilename = '/Users/stephen/Photos/image.png';
  *
  * image.createReadStream()
- *   .pipe(fs.createWriteStream('/Users/stephen/Photos/image.png'))
  *   .on('error', function(err) {})
  *   .on('response', function(response) {
  *     // Server connected and responded with the specified status and headers.
  *    })
- *   .on('complete', function() {
+ *   .on('finish', function() {
```

Comment:
Now that this event is attached to the gcloud stream, it actually is "complete". We could use a "finish" event on the fs stream below to tell readers that's when their file has been successfully written.